apps/storybook/src/TeamCreateOutputForm.stories.tsx

```diff
@@ -20,6 +20,13 @@ export const Normal = () => (
           }, 1000);
         })
       }
+      getAuthorSuggestions={() =>
+        new Promise((resolve) => {
+          setTimeout(() => {
+            resolve([{ label: 'author name', id: '1' }]);
```

Comment:
I caught this in my PR when i properly typed Multi Select. ID should read value.